w1r3/java/src/main/java/W1R3.java

```diff
@@ -318,9 +320,19 @@ private static Transport[] makeClients(String[] names) throws Exception {
   }
 
   private static class SingleShotUploader implements Uploader {
+    private static final BlobTargetOption[] BLOB_TARGET_OPTIONS =
+        new BlobTargetOption[] {
+          // set ifGenerationMatch=0 to allow retries
+          BlobTargetOption.doesNotExist(),
```

Comment:
We don't do this in other languages (well C++ does anyway for ${bad reasons}).  The rationale is that this is not a benchmark for retries.  If the upload fails, then no problem: we skip it and move on to the next iteration.

Your call, I think in practice there are so few of these that it makes no difference.

Reply:
Fair enough. That does make sense given how these benchmarks are run.